sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -276,8 +276,15 @@ shr(T1 src0, T2 src1, Sat sat = {}) {
 /// @return vector of rotated elements.
 template <typename T0, typename T1, int SZ>
 __ESIMD_API std::enable_if_t<
+#if defined(ESIMD_XE_HPC)
```

Comment:
ESIMD_XE_HPC macro is going to be eliminated from DPAS implementation soon. After that it would have no users.
IMO, we can allow int64 vectors without that macro. @kbobrovs - what do you think?